src/pkg/cli/client/byoc/aws/byoc_test.go

```diff
@@ -44,12 +48,20 @@ func TestDomainMultipleProjectSupport(t *testing.T) {
 		{"Project1", "tenant1", "web", port80, "web--80.project1.example.com", "web.project1.example.com", "web.project1.internal"},
 		{"Tenant2", "tenant1", "web", port80, "web--80.tenant2.example.com", "web.tenant2.example.com", "web.tenant2.internal"},
 		{"tenant1", "tenAnt1", "web", port80, "web--80.example.com", "web.example.com", "web.tenant1.internal"},
+		{"tenant1", "tenant1", "web", port80, "web--80.example.com", "web.example.com", "web.tenant1.internal"},
```

Comment:
same as previous?

Reply:
small difference in "tenAnt1" and "tenant1"